engine/src/main/java/io/camunda/zeebe/engine/state/instance/DbEventScopeInstanceState.java

```diff
@@ -53,7 +53,7 @@ public void shutdownInstance(final long eventScopeKey) {
     if (instance != null) {
       this.eventScopeKey.wrapLong(eventScopeKey);
       instance.setAccepting(false);
-      eventScopeInstanceColumnFamily.put(this.eventScopeKey, instance);
+      eventScopeInstanceColumnFamily.update(this.eventScopeKey, instance);
```

Comment:
@korthout Do we still need this method `shotdownInstance`? It is only used in tests. It also doesn't seem to be a helper method, but a method that was used in the past

Reply:
🤔 I'm not sure, but I think we can keep it out of scope.

It's clearly unused, but the value `isAccepting` is still used by some places to check that events should be triggered. We currently have [some bugs](https://github.com/camunda-cloud/zeebe/issues?q=is%3Aissue+is%3Aopen+eventsubprocess+label%3A%22Type%3A+Bug%22) related to triggering event sub process. Perhaps this is related, but I'm not sure. Perhaps this functionality can help in resolving those bugs. I think we can keep it out of scope of these changes for now